plugins/vfake.py

```diff
@@ -43,11 +47,16 @@ def setValuesFromFaker(col, faketype, rows):
     fakeMap[options.null_value] = options.null_value
 
     vd.addUndoSetValues([col], rows)
+    deterministic = options.vfake_deterministic
+    salt = options.vfake_seed_salt
```

Comment:
Let's use `vfake_salt` being non-empty to indicate this determinism, instead of having two options.

Reply:
Ok. Added a note in the option description and a comment in the logic to (hopefully) clarify things. I like the idea of one option, but I worry about it being a bit obscure and magical.

Reply:
Magical is the VisiData way!  Unless the options interact in some way other than one option enabling another option, it seems like it would also be confusing to set `vfake_seed_salt` and have it not do anything. I've even toyed with changing or removing all boolean options for just this reason.  (Case in point: we have `options.undo` because it may be a performance penalty, so what if it were instead the max number of rows that could be undone?  Instant configurable axis for very little effort.)

But it is your plugin, and so I ultimately defer to your taste.  You're the one that will be using it, after all :)